packages/table/src/Table.story.tsx

```diff
@@ -98,18 +101,17 @@ storiesOf('Table', module)
   .add('No nested Rows', () => (
     <Table
       data={defaultData}
-      columns={
-        <>
```

Comment:
Can I be super annoying and request we actually support fragments as well as an array? I mostly didn't want to deal with transforming a string into TableHeader.

Reply:
yes will revert

Reply:
done!